web/pgadmin/tools/schema_diff/__init__.py

```diff
@@ -750,17 +750,12 @@ def get_schemas(sid, did):
     This function will return the list of schemas for the specified
     server id and database id.
     """
```

Comment:
Any specific reason to remove exception handeling?

Reply:
There was one scenario given by Nikhil, which was basically giving issue in Browser tree schema collection object. The collection object was giving error on refresh of the node and Database connection gets disconnected, But Now if I test the same scenario, the issue is not reproducible and Also Nikhil confirmed that the issue is not reproducible after trying multiple times, so I am reverting to the earlier code.